.github/workflows/main.yml

```diff
@@ -95,6 +95,9 @@ jobs:
       uses: engineerd/setup-kind@v0.1.0
       with:
         version: "v0.6.1"
+      env:
+        ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
```

Comment:
Required to fix github actions see [here](https://github.com/actions/toolkit/issues/641#issuecomment-728963957)